SECURITY.md

```diff
@@ -2,10 +2,20 @@
 
 ## Reporting a Vulnerability
 
-To report security issues please send an e-mail to [security@lightprotocol.com](mailto:security@lightprotocol.com).
+We appreciate any vulnerability found and responsibly disclosed.
+All reward amounts are determined based on our ongoing [bug bounty program](https://immunefi.com/bug-bounty/light-protocol/information/) with Immunefi.
 
-For sensitive information or critical issues, please contact the above e-mail address with 'CRITICAL' in the subject line and we will respond with a mechanism to securely communicate.
+### Smart Contract and Infrastructure
 
+For smart contract and infrastructure vulnerabilities submission guidelines, please see our [bug bounty program](https://immunefi.com/bug-bounty/light-protocol/information/) with Immunefi.
+
+### Other vulnerabilities
+
+For vulnerabilities in any of our websites, email servers or other non-critical infrastructure, please send an e-mail to [security@lightprotocol.com](mailto:security@lightprotocol.com) referencing "Other Vulnerability" in the subject line.
 Please try to provide a clear description of any issue reported, along with how to reproduce the issue if possible.
 
-Bounty amounts are currently decided on a case by case basis.
+## Disqualification 
+
+You will be disqualified from receiving a reward for the following actions:
+- Filing a public ticket mentioning the vulnerability, or
+- Testing the vulnerability on the mainnet or testnet.
```

Comment:
I don't like this part it sounds like an unnecessary thread to me. We already have this covered in immunify.

Reply:
```suggestion
```